packages/core/src/api/tests/formatConversion/parse/getParseTestCases.ts

```diff
@@ -547,17 +461,113 @@ With Hard Break</p>
 </div>
 <p dir="ltr" style="line-height:1.38;margin-top:12pt;margin-bottom:12pt;"><span style="font-size:11pt;font-family:Arial,sans-serif;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Paragraph</span></p>
 </b>
-<br class="Apple-interchange-newline">`;
-
-    await parseHTMLAndCompareSnapshots(html, "parse-google-docs-html");
-  });
-
-  it("Parse codeblocks", async () => {
-    const html = `<pre><code>console.log("Should default to JS")</code></pre>
-    <pre><code data-language="typescript">console.log("Should parse TS from data-language")</code></pre>
-    <pre><code class="language-python">print("Should parse Python from language- class")</code></pre>
-    <pre><code class="language-ruby" data-language="typescript">console.log("Should prioritize TS from data-language over language- class")</code></pre>`;
-
-    await parseHTMLAndCompareSnapshots(html, "parse-codeblocks");
-  });
-});
+<br class="Apple-interchange-newline">`,
+  },
+  {
+    name: "codeBlocks",
+    conversionType: "html",
+    content: `<pre><code>console.log("Should default to JS")</code></pre>
+<pre><code data-language="typescript">console.log("Should parse TS from data-language")</code></pre>
+<pre><code class="language-python">print("Should parse Python from language- class")</code></pre>
+<pre><code class="language-ruby" data-language="typescript">console.log("Should prioritize TS from data-language over language- class")</code></pre>`,
+  },
+  {
```

Comment:
I get the idea, but I'm not entirely sure if it makes sense markdown / html of these cases should live in the same file

Reply:
Changed this so that tests for different formats are in separate arrays, and get their own `describe`. Imo we don't need to also separate to different files.